iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/agent/task/PipeTaskAgent.java

```diff
@@ -85,6 +85,24 @@ public PipeTaskAgent() {
     pipeTaskManager = new PipeTaskManager();
   }
 
+  ////////////////////////// PipeMeta Lock Control //////////////////////////
+
+  private void acquireReadLock() {
+    pipeMetaKeeper.acquireReadLock();
+  }
+
+  private void releaseReadLock() {
+    pipeMetaKeeper.releaseReadLock();
+  }
+
+  private void acquireWriteLock() {
+    pipeMetaKeeper.acquireWriteLock();
+  }
+
+  private void releaseWriteLock() {
+    pipeMetaKeeper.releaseWriteLock();
+  }
+
   ////////////////////////// Pipe Task Management Entry //////////////////////////
 
   public synchronized List<TPushPipeMetaRespExceptionMessage> handlePipeMetaChanges(
```

Comment:
```suggestion
public synchronized List<TPushPipeMetaRespExceptionMessage> handlePipeMetaChanges(..) {
  lock();
  try {
    handlePipeMetaChangesWithoutLock(...); // private, without synchronized, without r/w lock
  } finally {
    unlock();
  }
}
```

It seems that synchronized can be removed

Reply:
You can consider to use this method to simplify the other methods in this class.